acceptance/openstack/clustering/v1/autoscaling_test.go

```diff
@@ -0,0 +1,62 @@
+package v1
```

Comment:
I was thinking about grouping all these(profiles, clusters, nodes, receivers...)  into a single acceptance test called autoscaling_test.go, which of the following should I use 
1) +build acceptance clustering autoscaling
2) +build acceptance clustering profiles clusters nodes receivers
3) +build acceptance clustering autoscaling profiles clusters nodes receivers

Reply:
It actually doesn't matter. The important piece is the `+build acceptance` part. And the blank line after.

The other tags are more for documentation purposes. The third tag (clustering) in this case describes the service being tested. Anything else is purely extra data.

The reason it doesn't matter is twofold:

1. When CI triggers the acceptance tests, the tests are only based on the word "acceptance". This is enough to filter unit tests from acceptance tests. 
2. Extra conditions can be added for each test (or service suite) to filter out a service by way of environment variables: https://github.com/gophercloud/gophercloud/blob/master/acceptance/clients/conditions.go The reason this is a better way to fine-tune test filtering is because build tags are per file whereas environment variables can perform filtering on each test. This way we don't need, for example, a test file of `identity_test.go` and `identity_adminonly_test.go`.